src/arrayMethodSort.js

```diff
@@ -6,6 +6,41 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
create default function to use instead of `if` condition
So this function will invoke the same as you did with `compareFunction `

```
  const compareFunctionByDefault = (a, b) => {
    const stringA = String(a);
    const stringB = String(b);

    if (stringA > stringB) {
      return 1;
    } 
    
    if (stringA === stringB) {
      return 0;
    } 
    
    return -1;
  };

```

```suggestion
  [].__proto__.sort2 = function(compareFunction = compareFunctionByDefault) {
```